src/main/java/org/benetech/servicenet/service/impl/ActivityServiceImpl.java

```diff
@@ -151,4 +172,13 @@ private ActivityDTO getEntityActivity(ActivityInfo info, Map<UUID, ExclusionsCon
             return new PageImpl<>(Collections.emptyList(), pageable, Collections.emptyList().size());
         }
     }
+
+    private ProviderActivityRecordDTO mapToProviderActivityRecordDTO(Organization organization) {
+        try {
+            Optional<ActivityRecordDTO> opt = recordsService.getRecordFromOrganization(organization);
```

Comment:
is it possible to only get necessary data from the database instead of mapping activity records? or would it take too much effort?